package.json

```diff
@@ -11,13 +11,13 @@
     "test": "node packages/react-scripts/scripts/test.js --env=jsdom"
   },
   "devDependencies": {
-    "babel-eslint": "6.1.2",
-    "eslint": "3.5.0",
-    "eslint-config-react-app": "0.2.1",
-    "eslint-plugin-flowtype": "2.18.1",
-    "eslint-plugin-import": "1.12.0",
-    "eslint-plugin-jsx-a11y": "2.2.2",
-    "eslint-plugin-react": "6.3.0",
+    "babel-eslint": "^7.1.1",
+    "eslint": "^3.15.0",
+    "eslint-config-react-app": "0.5.1",
+    "eslint-plugin-flowtype": "^2.30.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "2.2.3",
```

Comment:
Our CI currently runs lint in `e2e-simple`, see https://travis-ci.org/facebookincubator/create-react-app/jobs/201147269#L204-L234. We don't treat warnings as failures though, would you like to?